### DISCUSSION TITLE
Streaming device support (Android TV / Apple TV)

### DISCUSSION THREAD

**Discussion by GrabbenD** [jellyfin/jellyfin-vue#2148] [Questions] [answered]
I know you guys have been experimenting with desktop apps using [Electron](https://github.com/jellyfin/jellyfin-vue/pull/924) and [Tauri+libmpv](https://github.com/jellyfin/jellyfin-vue/pull/925).

How about Android TV and Apple TV? Are there any plans or work around supporting streaming devices such as Nvidia Shield, Chromecast, FireTV devices or Apple TV?

**Comment by ferferga** [comment, marked as answer]
Right now no. There's already a lot of things to tackle to get this client to the point I want it to be and nowadays it's basically me keeping the garden.

The only idea that's on my roadmap related to this it's to improve gamepad/remote control navigation, but still that means that you'll need a browser in your device.

I want first to ensure a proper experience for all the device types and ranges with the "vanilla" client, and once that's done, investigate ports to other platforms or even "plugins" #2002
